pkg/serverless/logs/testdata/platform_incorrect_runtime_done_log_valid.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think this file name might be wrong -- it says "valid" but it also says "incorrect"